indexer/pkg/readers/verifier_reader.go

```diff
@@ -85,7 +85,7 @@ func (v *verifierReader) ProcessMessage(messageID protocol.Bytes32) (chan common
 // Start returns immediately after spawning the background goroutine. It does not
 // wait for the goroutine to complete.
 func (v *verifierReader) Start(ctx context.Context) error {
-	runCtx, cancel := context.WithCancel(ctx)
+	runCtx, cancel := context.WithCancel(context.Background())
```

Comment:
I'm not sure the indexer is following the same Start/Close pattern as the verifier and executor - can we exclude from this PR?